collie/metrics.py

```diff
@@ -15,83 +15,69 @@
 from collie.model import BasePipeline
 
 
-def _get_user_item_pairs(model: BasePipeline,
-                         user_or_item_ids: Union[np.array, torch.tensor],
-                         n_items_or_users: int,
-                         device: Union[str, torch.device]) -> Tuple[torch.tensor, torch.tensor]:
+def _get_row_col_pairs(row_ids: Union[np.array, torch.tensor],
+                       n_cols: int,
+                       device: Union[str, torch.device]) -> Tuple[torch.tensor, torch.tensor]:
     """
-    Create tensors pairing each input user ID with each item ID or vice versa.
+    Create tensors pairing each input row ID with each column ID.
 
     Parameters
     ----------
-    model: collie.model.BasePipeline
-        Model that can take a (user_id, item_id) pair as input and return a recommendation score
-    user_or_item_ids: np.array or torch.tensor, 1-d
+    row_ids: np.array or torch.tensor, 1-d
         Iterable[int] of users or items to score
-    n_items_or_users: int
+    n_cols: int
         Number of items or users in the training data
     device: string
         Device to store tensors on
 
     Returns
     -------
-    users: torch.tensor, 1-d
-        Tensor with ``n_items_or_users`` copies of each user ID
-    items: torch.tensor, 1-d
-        Tensor with ``len(user_or_item_ids)`` copies of each item ID
+    rows: torch.tensor, 1-d
+        Tensor with ``n_cols`` copies of each row ID
+    cols: torch.tensor, 1-d
+        Tensor with ``len(row_ids)`` copies of each column ID
 
     Example
     -------
     .. code-block:: python
 
-        >>> user_ids = np.array([10, 11, 12])
-        >>> n_items = 4
-        >>> user, item = _get_user_item_pairs(user_ids: user_ids, n_items: 4, device: 'cpu'):
-        >>> user
+        >>> row_ids = np.array([10, 11, 12])
+        >>> n_cols = 4
```

Comment:
Maybe make this example a little clearer with comments: 

```suggestion
        >>> # let's assume an interactions matrix with users as rows and item as columns
        >>> row_ids = np.array([10, 11, 12])  # user IDs 
        >>> n_cols = 4  # number of items
```